examples/PagerNode/Sample/ViewController.m

```diff
@@ -47,11 +63,13 @@ - (NSInteger)numberOfPagesInPagerNode:(ASPagerNode *)pagerNode
   return 5;
 }
 
-- (ASCellNode *)pagerNode:(ASPagerNode *)pagerNode nodeAtIndex:(NSInteger)index
+- (ASCellNodeBlock)pagerNode:(ASPagerNode *)pagerNode nodeBlockAtIndex:(NSInteger)index
 {
-  PageNode *page = [[PageNode alloc] init];
-  page.backgroundColor = [UIColor blueColor];
-  return page;
```

Comment:
For sample code, we should add a comment above the block saying something like (feel free to edit or reduce):

// NOTE: For implementers of this method, it is critical to use the value of "index" outside of the block to access any data source objects, etc.
// The block is executed concurrently in the background and data source values may change at a given index.